src/components/registration/RegistrationTable.jsx

```diff
@@ -1,21 +1,24 @@
 import React, { useContext } from "react";
 import MUIDataTable from "mui-datatables";
 import PropTypes from "prop-types";
+import { Typography } from "@material-ui/core";
 
 import { FieldsContext } from "../../context/fields";
 import RegistrationTableColumns from "../../constants/registration/RegistrationTableColumns";
 import QuestionTypes from "../../constants/QuestionTypes";
 
 const options = {
   responsive: "standard",
+  rowsPerPage: 25,
+  rowsPerPageOptions: [25, 50, 100],
   selectableRows: "none",
 };
 
-function getTableRows(families, extraFields) {
-  if (extraFields.length) {
+function getTableRows(families, extraColumns) {
+  if (extraColumns.length) {
```

Comment:
Why was this renamed?

Reply:
ooooops I didn't mean to rename that one 😅 good catch